Makefile

```diff
@@ -78,6 +78,10 @@ stack-yaml:
 
 # Overwrite existing golden files
 accept-golden:
+	rm -rf test/examples-golden/positive
+	rm -rf test/examples-golden/negative
+	# If we want further commands, put - at the start, to ignore
+	# the error from this command
 	stack test --test-arguments "--accept"
```

Comment:
Do you have any reproducible error?

Reply:
Not any easy ones. This mainly comes out of when we refactor tests that the  `stack test --test-arguments "--accept"` line alone does not cause already written files to always be overwritten.

Also I think this has the side benefit of catching when we delete some tests like `.circuit`, as they won't be re-generated

Reply:
Yes, I can see its benefits 👍